### ISSUE TITLE
How can I find the meaning of the A codes in the DF toy dataset? For instance, A0, A1 ...

### ISSUE THREAD

**Issue by zahrakhanjani128** [asvspoof-challenge/2021#20]

**Comment by TonyWangX** [comment]
Hi, do you mean the labels in the protocol.txt file?

The labels A01 and so on are the IDs of the speech synthesis systems (either text-to-speech or voice conversion). 

These systems are described in https://doi.org/10.1016/j.csl.2020.101114
(or Arxiv version: https://arxiv.org/abs/1911.01601)

Note:
1. The ASVspoof2021 DF task requires the participants to use the ASVspoof2019 LA training set to train their models. All the labels of the speech synthesis systems are described in the above document. The toy dataset is a small portion of the ASVspoof2019 LA dataset. 
2. The ASVspoof2021 DF task evaluation data contains not only data from the ASVspoof2019 LA test set but also data from other sources. The evaluation condition is also more complicated, and the labels of the evaluation data are enriched.  Please check this: https://github.com/asvspoof-challenge/2021/tree/main/eval-package

**Comment by zahrakhanjani128** [comment]
Great! Thanks for the clarification
